### PR TITLE
Create drag and drop wizard documentation

### LINKED ISSUE #642: Drag and Drop in Dashboards
What do you want to do?

- Make a change to existing documentation
**- Add new documentation**
- Report a technical problem with the documentation
- Other

Please provide the following information:

**Give a brief description of your request.** 
Document new functionality coming in 2.1: Drag and Drop enhancements to Dashboards.

**Which versions are affected?**
New to 2.1

**Which pages or URLs are affected?**
We have limited information on our Dashboards in the doc, but the info on dashboards starts here: https://opensearch.org/docs/latest/dashboards/index/ 

**What resources are available** (for example, links to a related issue, PR, design doc, or feature brief)?
https://github.com/opensearch-project/OpenSearch-Dashboards/issues/1157
https://github.com/opensearch-project/OpenSearch-Dashboards/issues/379

**Who should we contact with technical questions?**
Andrew Hopp is the PM for Dashboards; Sean Neumann is Engineering manager of Dashboards

**Is there anything else you’d like us to know about your requested change?**

### PR DESCRIPTION
**Pull request by vagimeli**
Signed-off-by: vagimeli <vagimeli@amazon.com>

### Description
Creates initial drag and drop wizard documentation for experimental deature

### Issues Resolved
#642  

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _dashboards/drag-drop-wizard.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
+layout: default
+title: Drag and drop wizard (experimental feature)
+nav_order: 15
+---
+
+# Build visualizations by  dragging and dropping fields
```
Comment (alicejw-aws) [inline _dashboards/drag-drop-wizard.md:7]: remove extra space

File: _dashboards/drag-drop-wizard.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+---
+layout: default
+title: Drag and drop wizard (experimental feature)
```
Comment (alicejw-aws) [inline _dashboards/drag-drop-wizard.md:3]: suggest to keep visualization type names parallel in left nav: 
(e.g. Region map visualizations) 
maybe change title to "Wizard visualizations (experimental feature)"

File: _dashboards/drag-drop-wizard.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+nav_order: 15
```
Comment (alicejw-aws) [inline _dashboards/drag-drop-wizard.md:4]: might change number to follow other visualization item at 7
nav_order: 8

File: _dashboards/drag-drop-wizard.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+You can use the drag and drop visualization Wizard in OpenSearch Dashboards to create your data visualizations easily with just a few clicks or a single mouse gesture. 
+
+The drag and drop visualization Wizard:
+
+* Gives you immediate an view of your data without the need to preselect the visualization output 
+* Gives you the flexibility to change visualization types and index patterns on the fly
+* Gives you the ability to easily navigate between multiple screens 
+
+Beginning with OpenSearch release 1.3, the drag and drop wizard is available as an experimental feature. We invite you to try the wizard and let us know how you feel about it.
```
Comment (alicejw-aws) [inline _dashboards/drag-drop-wizard.md:17]:
```suggestion
Beginning with OpenSearch release 1.3, the drag and drop wizard is available as an experimental feature. We invite you to try the wizard and let us know how you feel about it.
{: .note}
```

Comment (alicejw-aws) [inline _dashboards/drag-drop-wizard.md:17]: add this note syntax directly on next line below sentence to create a callout:
{: .note}

File: _dashboards/drag-drop-wizard.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
+---
+layout: default
+title: Drag and drop wizard (experimental feature)
+nav_order: 15
+---
+
+# Build visualizations by  dragging and dropping fields
+
+You can use the drag and drop visualization Wizard in OpenSearch Dashboards to create your data visualizations easily with just a few clicks or a single mouse gesture.
```
Comment (hdhalter) [inline _dashboards/drag-drop-wizard.md:9]: lowercase 'w' on wizard

File: _dashboards/drag-drop-wizard.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+layout: default
+title: Drag and drop wizard (experimental feature)
+nav_order: 15
+---
+
+# Build visualizations by  dragging and dropping fields
+
+You can use the drag and drop visualization Wizard in OpenSearch Dashboards to create your data visualizations easily with just a few clicks or a single mouse gesture. 
+
+The drag and drop visualization Wizard:
```
Comment (hdhalter) [inline _dashboards/drag-drop-wizard.md:11]: lowercase 'w' on wizard

**Review by hdhalter — APPROVED** [review]
Just a couple small things. Great job!

File: _dashboards/drag-drop-wizard.md (hunk 5 above)
Comment (alicejw-aws) [reply _dashboards/drag-drop-wizard.md:9]: it's uppercase in the UI, so text should match it

File: _dashboards/drag-drop-wizard.md [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+
+1. Open [OpenSearch Dashboards playground](https://playground.opensearch.org/app/home#/).
+2. Go to **Management** **>** **Stack Management** **>** **Advanced Settings**.
+<img src="{{site.url}}{{site.baseurl}}/images/stack-managment-settings.png" alt="Stack management screenshot">
+1. Go to **Visualization** and turn on **Enable experimental visualizations**.
+<img src="{{site.url}}{{site.baseurl}}/images/enable-experimental-viz.png" alt="Enable experimental visualizations screenshot">
+
+# Try out the drag and drop wizard or create a new visualization using Wizard
+
+You can explore the drag and drop wizard using the [OpenSearch Wizard playground](https://playground.opensearch.org/app/wizard), or create a new visualization using the Wizard panel.
```
Comment (alicejw-aws) [inline _dashboards/drag-drop-wizard.md:35]: when they go to the playground URL, that's the Wizard "panel"
create visualization -> select Wizard 

maybe reorg sentence as follows: 
You can explore the drag and drop Wizard panel using the [OpenSearch Wizard playground](https://playground.opensearch.org/app/wizard), or create a new visualization and select the Wizard type.

**Review by alicejw-aws — APPROVED** [review]
Looks good! made a couple of suggestions

File: _dashboards/drag-drop-wizard.md (hunk 7 above)
Comment (alicejw-aws) [inline _dashboards/drag-drop-wizard.md:35]: when they go to the playground URL, that's the Wizard "panel"
but when they do create visualization -> they select Wizard from the viz types available

maybe reorg sentence as follows: 
You can explore the drag and drop Wizard directly from the [OpenSearch Wizard playground](https://playground.opensearch.org/app/wizard), or you can create a new visualization and select Wizard.

File: _dashboards/drag-drop-wizard.md [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+
+To experiment with the drag and drop wizard, you first need to enable experimental visualizations in your OpenSearch playground:
+
+1. Open [OpenSearch Dashboards playground](https://playground.opensearch.org/app/home#/).
+2. Go to **Management** **>** **Stack Management** **>** **Advanced Settings**.
+<img src="{{site.url}}{{site.baseurl}}/images/stack-managment-settings.png" alt="Stack management screenshot">
+1. Go to **Visualization** and turn on **Enable experimental visualizations**.
+<img src="{{site.url}}{{site.baseurl}}/images/enable-experimental-viz.png" alt="Enable experimental visualizations screenshot">
+
+# Try out the drag and drop wizard or create a new visualization using Wizard
```
Comment (alicejw-aws) [inline _dashboards/drag-drop-wizard.md:33]: suggestion:
"Try out the drag and drop Wizard in the playground or create a new visualization Wizard."

File: _dashboards/drag-drop-wizard.md (hunk 5 above)
Comment (vagimeli) [reply _dashboards/drag-drop-wizard.md:9]: Name will change in release 2.4, so we'll update naming convention again

File: _dashboards/drag-drop-wizard.md (hunk 7 above)
Comment (vagimeli) [reply _dashboards/drag-drop-wizard.md:35]: Great rewrite, accepted.

File: _dashboards/drag-drop-wizard.md (hunk 8 above)
Comment (vagimeli) [reply _dashboards/drag-drop-wizard.md:33]: Accepted suggested rewrite

**Pull request merged**